go/base/management/commands/go_manage_routing_table.py

```diff
@@ -51,48 +43,46 @@ def handle(self, *args, **options):
                 else:
                     raise CommandError('Please provide %s:' % (opt.dest,))
 
-        command_options = [c for c in self.CONFLICTING_OPTIONS if options[c]]
-        if len(command_options) != 1:
+        # Ensure a single option is chosen from the local options
+        cmd_options = [c.dest for c in self.LOCAL_OPTIONS if options[c.dest]]
+        if len(cmd_options) != 1:
             raise CommandError('Please provide exactly one of: %s' % (
-                ['--%s' % c for c in self.CONFLICTING_OPTIONS],))
-
-        user = get_user_by_email(options['email-address'])
-        user_api = vumi_api_for_user(user)
+                ['--%s' % c.dest for c in self.LOCAL_OPTIONS],))
```

Comment:
The problem with the old way of doing this is that `CONFLICTING_OPTIONS` was a separate list of things that has to match a subset of the things in `LOCAL_OPTIONS`. The problem with the new way is that there's no distinction between options that conflict and options that don't.

What about renaming `LOCAL_OPTIONS` to `COMMAND_OPTIONS` and having a separate (empty) `LOCAL_OPTIONS` as well?